python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -380,3 +390,109 @@ def close(self) -> None:
         if self.is_local_node(self._reader_node_id):
             self.ensure_registered_as_reader()
         self._worker.core_worker.experimental_channel_set_error(self._reader_ref)
+
+
+@PublicAPI(stability="alpha")
+class MultiChannel(ChannelInterface):
+    def __init__(
+        self,
+        writer: Optional[ray.actor.ActorHandle],
+        readers: List[Optional[ray.actor.ActorHandle]],
+        _channel_id: Optional[str] = None,
+        _local_channel: Optional[IntraProcessChannel] = None,
+        _remote_channel: Optional[Channel] = None,
+        _channels: List[Channel] = [],
```

Comment:
Optional[List[Channel]] = None

If we initialize the arg with an object, it will tight to the class definition. Premetive type is fine.